tests/Selenium/helpers/checkout.js

```diff
@@ -5,7 +5,7 @@ import fixtures from '../fixtures';
 export function addSomeProductToCart () {
     return browser
         .url(fixtures.urls.aCategoryWithProducts)
-        .click('.product-miniature:nth-of-type(2) a.product-thumbnail')
+        .click('.products :nth-of-type(2) a.product-thumbnail')
```

Comment:
be careful, I believe you have an extra space in your selector

Reply:
@jolelievre No you can see the comment of @PierreRambaud here : https://github.com/PrestaShop/PrestaShop/pull/14558#discussion_r302036299

Reply:
Hi, I want to update this file, but where exactly is it. I have no folder test in my PS 1.7.1

Reply:
The `tests` folder is not a part of the final ZIP build. If you want to contribute to the tests, you need to clone the repository.